app/controllers/collections_controller.rb

```diff
@@ -13,6 +13,8 @@ def new
 
     @form = CollectionForm.new(collection)
     @form.prepopulate!
+
+    add_breadcrumb 'Create', ''
```

Comment:
I guess the empty string has to be passed because of l35 in the application controller?

Reply:
yes, but this could be done more elegantly by perhaps passing in a hash to the `add_breadcrumb` method, allowing us to see more clearly what is being passed in and eliminating that empty value...i'll play with that, it may make the definition of the `add_breadcrumb` method more clear as well (right now just a copypasta from the gist you sent)

Reply:
That approach sounds better to me too, @peetucket

Reply:
Refactored and it looks a lot cleaner now.  Only deal is you add one breadcrumb at a time instead of being able to pass in an array, but I think this makes for cleaner method calls anyway.